packages/block-library/src/columns/style.scss

```diff
@@ -99,6 +102,42 @@
 			}
 		}
 	}
+
+	// Divided block style.
+	//
+	// Following styles update the display of borders for this block style. They
+	// require the use of !important to override the inline styles supplied by
+	// the border block support allowing user defined borders.
+	&.is-style-divided .wp-block-column {
+		// Default divided border to use until user specifies something different.
+		border: 1px solid $gray-900;
```

Comment:
If you replace this with `currentColor`, not only will it work in dark themes, but it provides a meaningful default by sharing color with the surrounding text until the user makes a choice.

Reply:
Done ✅ 

This has been updated just in case this PR doesn't get abandoned in favour of reworking block supports.